src/main/webapp/js/instructorFeedbackEdit/questionRubric.es6

```diff
@@ -118,12 +109,44 @@ function addRubricCol(questionNum) {
         $(rubricRowFragment).insertAfter(lastTd);
     }
 
+    // Add options row at the end
+    const optionsRow =
+        `<td class="align-center rubricCol-${questionNum}-${newColNumber - 1}" data-col="${newColNumber - 1}">
+            <div class="btn-group">
+                <button type="button" class="btn btn-default" id="rubricMoveChoiceLink-${questionNum}-${newColNumber - 1}-l"
+                    onclick="swapRubricCol(${newColNumber - 1}, ${questionNum}, true)" data-toggle="tooltip"
+                    data-placement="top" title="Move column left">
+                    <span class="glyphicon glyphicon-arrow-left"></span>
+                </button>
+                <button type="button" class="btn btn-default" id="rubricRemoveChoiceLink-${questionNum}-${newColNumber - 1}"
+                    onclick="removeRubricCol(${newColNumber - 1}, ${questionNum})"
+                    onmouseover="highlightRubricCol(${newColNumber - 1}, ${questionNum}, true)"
+                    onmouseout="highlightRubricCol(${newColNumber - 1}, ${questionNum}, false)">
+                    <span class="glyphicon glyphicon-remove"></span>
+                </button>
+                <button type="button" class="btn btn-default" id="rubricMoveChoiceLink-${questionNum}-${newColNumber - 1}-r"
+                    onclick="swapRubricCol(${newColNumber - 1}, ${questionNum}, false)" data-toggle="tooltip"
+                    data-placement="top" title="Move column right">
```

Comment:
Yes, it's preferred to not do this (though in this case it's not as bad since `swapRubricCol` is in the same file). Put the `onclick` in a separate function.